src/features/presence/PresenceCollaborators.scss

```diff
@@ -0,0 +1,45 @@
+@import './styles';
+
+.bdl-PresenceCollaborators-item {
+    display: flex;
+    align-items: center;
+    margin-top: 10px;
+
+    &.hover {
```

Comment:
We should use SUIT naming conventions whenever possible.

Reply:
Updated! Now using a `is-not-hovered` class and converted `.hover` to use the pseudo-class